gulpfile.js

```diff
@@ -127,6 +127,16 @@ gulp.task('copy', ['less'], function() {
 });
 
 /**
+ * Deploy dist to static
+ */
+gulp.task('deploy', ['copy', 'dist'], function() {
```

Comment:
@bhomnick did you mean to require the `copy` task before the `dist` task here? Seems to be the opposite below in the `default` task.